lib/elrondex/sc/pair_sc.ex

```diff
@@ -1,6 +1,17 @@
 defmodule Elrondex.Sc.PairSc do
   alias Elrondex.{Sc, Transaction, Account, REST, ESDT, Pair, Network}
 
+  @doc """
+  Swaps token1 amount for a minimum of token2 amount.
```

Comment:
Swaps a certain token amount for a minimum of another token amount.

I think it would be better like this.